README.md

```diff
@@ -24,6 +24,7 @@
 > Describe how to install, build, and test your app
 
 ## Contributors
-- [Zsolt Meszaros](https://github.com/zsoltime)
-- [Aimen Aounallah](https://github.com/Mizou9999)
 
+-  [Zsolt Meszaros](https://github.com/zsoltime)
+-  [Aimen Aounallah](https://github.com/Mizou9999)
+   > > > > > > > 30a5802fa5c26e9d3158f58ae293bd84e44984c4
```

Comment:
And it seems like some leftover from a merge conflict. You can remove this line, too.